api/app/views/spree/api/variants/small.v1.rabl

```diff
@@ -14,4 +14,4 @@ child :option_values => :option_values do
   attributes *option_value_attributes
 end
 
-child(:images => :images) { extends "spree/api/images/show" }
+child(:display_images => :images) { extends "spree/api/images/show" }
```

Comment:
I think @athal7 had concerns (rightly so) about the performance impact of this. Has the admin been tested with a large dataset loaded?